saleor/product/migrations/0130_create_product_description_search_vector.py

```diff
@@ -0,0 +1,93 @@
+# Generated by Django 3.1.5 on 2021-01-21 09:19
+
+import django.contrib.postgres.indexes
+import django.contrib.postgres.search
+from django.db import migrations, models
+
+
+def parse_draftjs_content_to_string(definitions):
+    string = ""
+    blocks = definitions.get("blocks")
+    if not blocks or not isinstance(blocks, list):
```

Comment:
```
if not isinstance(blocks, list):
```
is enough

Reply:
It's code from 2.11, I would like to keep it the same at it was on 2.11